src/common/client-app.ts

```diff
@@ -69,8 +74,9 @@ export class App {
 
   onInterval= () => {
     var current = this.boardVM.toPlain();
-    var myChanges = rfc6902.createPatch(this.shadow, current);
-    if (myChanges.length) {
+    var myChanges = rfc6902.createPatch(this.shadow, current);
+    if (myChanges.length) {
+      rfc6902.applyPatch(this.shadow, utils.clone(myChanges));
```

Comment:
(We was talked about it but I will at for the record)

In my opinion this clone is not necessary because `myChanges` is already a new object.

I thin k that we have two possible approaches: 

* to always clone, in that case we should use a wrapper to reduce code duplication and do not forget to clone
* clone only when it is really need, in that case we should not clone when it is not necessary (in this case, probably you should also improve test infrastructure to clone on client/server communication to simulate it in a more realistic way)

I am fine with any of these approaches, but I prefer to not have an intermediate state because it is not optimal nor simpler.